webpack/webpack.config.base.js

```diff
@@ -117,8 +117,10 @@ export const config = {
       vm: require.resolve('vm-browserify'),
       assert: require.resolve('assert'),
       'lottie-web': path.resolve('node_modules/lottie-web/build/player/lottie_light.js'),
-      fs: false,
-      path: false,
+      fs: require.resolve('browserify-fs'),
+      path: require.resolve('path-browserify'),
+      os: require.resolve('os-browserify/browser'),
+      process: require.resolve('process/browser'),
```

Comment:
Guess Bitflow isn't the best authored lib if we have to add tonnes of node polyfills 🤦🏼